actor-server/actor-core/src/main/scala/im/actor/server/api/TypeMappers.scala

```diff
@@ -41,6 +42,18 @@ private[api] trait MessageMapper {
     ByteString.copyFrom(message.toByteArray)
   }
 
+  private def applyInetAddress(remoteAddr: String): InetAddress = {
+    if (remoteAddr != null) {
+      InetAddress.getByName(remoteAddr)
+    } else {
+      null
+    }
+  }
+
+  private def unapplyInetAddress(address: InetAddress): String = {
```

Comment:
I see. You mean I use getByAddress method with byte array input? Only this way there is no any need for conversion in my code.

`InetAddress.getByAddress(bytes)`